project/.travis.yml.twig

```diff
@@ -41,7 +41,7 @@ env:
     - SYMFONY_DEPRECATIONS_HELPER=max[self]=0
     - TARGET=test
     - UPSTREAM_URL=https://github.com/sonata-project/{{ repository_name }}.git
-    - PHPUNIT_VERSION={{ phpunit_version|default('7') }}
+    - PHPUNIT_VERSION={{ phpunit_version|default('8') }}
```

Comment:
Can you remove the old `phpunit_version` settings from the project configuration?